### DISCUSSION TITLE
How to add custom comment for test result both passed and failed

### DISCUSSION THREAD

**Discussion by eznxxy** [PeterNgTr/codeceptjs-testrail#85] [Q&A] [answered]
i want custom my comment for passed and failed result, 

for example:

      testCase: {
        passed: {
          status_id: 1,
          comment: 'Custom comment goes here',
        },
        failed: { 
          status_id: 5,
          comment: 'Custom comment goes here',
        },
      },

any idea how to do that?

**Comment by kobenguyent** [comment, marked as answer]
hey @eznxxy please try the package `codeceptjs-testrail (1.10.0) `

**Reply by eznxxy** [reply]
Thank you!
